extensions/bitwarden/src/hooks.ts

```diff
@@ -0,0 +1,52 @@
+import { getLocalStorageItem, showToast, ToastStyle, removeLocalStorageItem, setLocalStorageItem } from "@raycast/api";
+import execa from "execa";
+import { useState, useEffect } from "react";
+import { getWorkflowEnv } from "./utils";
+
+const LocalStorageKey = "sessionToken"
+export function useSessionToken(): [string | null | undefined, (sessionToken: string | null) => void] {
```

Comment:
I agree, however I am not sure it will work quite right with react logic. I cannot really generate the form on demand, especially in a useEffect call.

Reply:
I think your proposition does not play well with react flow and rerendering logic. I'll keep my implementation for the first release.